pymilvus/orm/collection.py

```diff
@@ -156,6 +156,25 @@ def __init__(
         self._schema_dict = self._schema.to_dict()
         self._schema_dict["consistency_level"] = self._consistency_level
 
+        self._normalization_fields = kwargs.get("normalization_fields", None)
+        if self._normalization_fields:
+            self._vector_fields = self._get_vector_fields()
+            if self._normalization_fields == 'all':
+                self._normalization_fields = self._vector_fields
+            elif isinstance(self._normalization_fields, list):
+                for field in self._normalization_fields:
+                    if field not in self._vector_fields:
+                        raise BaseException(ExceptionsMessage.InvalidVectorFields % (field, ', '.join(self._vector_fields)))
+            else:
+                raise BaseException(ExceptionsMessage.InvalidNormalizationParam % (self._normalization_fields))
```

Comment:
ditto

Reply:
@XuanYang-cn  Thank for your feedback. I have made the necessary changes based on your comments.